File: JavaScript/merge.js

nums1 = [1,2,3,0,0,0], m = 3, nums2 = [2,5,6], n = 3

let merge = (nums1, m, nums2, n) => {
    let arr1 = []
    let arr2 = []
    
    for (let x = 0; x < m; x++) {
        arr1.push(nums1[x])
    }
    for (let x = 0; x < n; x++) {
        arr2.push(nums2[x])
    }
    let newArr = ([...arr1, ...arr2]).sort((a, b) => a - b)
 
   nums1.splice(0, (m + n), ...newArr);
 return nums1
}
    
console.log(merge(nums1, m, nums2, n))

File: Python/merge.py

nums1 = [1,2,3,0,0,0] 
m = 3
nums2 = [2,5,6]
n = 3

def merge(nums1, m, nums2, n):
  
    arr1 = []
    arr2 = []

    for x in range(0, m, 1):
        arr1.append(nums1[x])
    for y in range(0, n, 1):
        arr2.append(nums2[y])
    nums1.clear()
    nums1.extend(sorted(arr1 + arr2))

    return nums1


print(merge(nums1, m, nums2, n))